src/python/doc/delaunay_complex_ref.rst

```diff
@@ -0,0 +1,29 @@
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+=================================
+Delaunay complex reference manual
+=================================
+
+.. autoclass:: gudhi.DelaunayComplex
+   :members:
+   :undoc-members:
```

Comment:
On d2702adb2, I removed `:undoc-members:`, but it does not help to remove `__new__(**kwargs)`.
I also tried to give an explicit list of members to document (cf. [autoclass doc](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoclass)), but it des not work too.
This behaviour seems to happen only with cython modules, cf. [gudhi __new__ methods](https://gudhi.inria.fr/python/latest/search.html?q=__new__)

Reply:
It looks like there is something strange (bug?) in the interaction between sphinx and cython. Even if I remove `:members:` I can't get rid of the doc of `__init__` and `__new__` :confused:.
But that doesn't seem new, we never had to say `:special-members:` for cython classes, unlike pure python. So let's ignore it for now.

If we want to clean up a bit in a separate PR (including the other files), we can use `:exclude-members: __new__`.
We could also use `autodoc_default_options` to avoid having to specify `:members:` all the time. Avoid `:show-inheritance:` when the only base is `object`. etc.

Reply:
Oh great, `:exclude-members: __new__` is working ! I open issues.